### ISSUE TITLE
vis_app showing poor result even though training metrics look great

### ISSUE THREAD

**Issue by jackkwok** [ClementPinard/Pytorch-SiamFC#10]
Very nice work!

I trained the default model with default params for 60 epoch and got center_error even slightly better than reported on the README.  My result:
{
    "AUC": 0.9630974820435345,
    "center_error": 10.192100637383383,
    "loss": 0.07371289491727948
}

However, when I run vis_app with my best trained model weights, the results look very bad (tried on both train and val data).  The center_error is huge and the score map heatmap looks uniformly pink for the whole image:

![siamfc_visapp](https://user-images.githubusercontent.com/1870448/50524480-c1851600-0a8a-11e9-92a2-ce9b34b08cd2.png)

When I run vis_app with the provided BaselinePretrained.pth.tar, the results look great.

Is there a bug?

**Comment by ClementPinard** [comment]
looks like a batchnorm problem if you ask me !
